pkg/queue/sharedmain/main.go

```diff
@@ -86,7 +86,7 @@ type config struct {
 	RevisionIdleTimeoutSeconds          int    `split_words:"true"` // optional
 	ServingReadinessProbe               string `split_words:"true"` // optional
 	EnableProfiling                     bool   `split_words:"true"` // optional
-	EnableHTTP2AutoDetection            bool   `split_words:"true"` // optional
+	EnableHTTPAutoDetection             bool   `split_words:"true"` // optional
```

Comment:
Poked at the library and there is a simpler fix - https://go.dev/play/p/7J5IfxokhKk

```suggestion
	EnableHTTP2AutoDetection             bool   `envconfig:"ENABLE_HTTP2_AUTO_DETECTION"` // optional
```

Then you can undo all the other changes

Reply:
Cool I will update the PR. In addition, people should know that `split_words` is not working as excepted there and maybe we should be explicit for the other vars too?